<launch>


	<include file = "$(find robot_setup_tf)/launch/octomap_mapping.launch" />


	<node pkg="robot_setup_tf" type="octomap_to_spherical" name="octomap_to_spherical" output="screen" />


</launch>
